### DISCUSSION TITLE
음량 관련 문의 드립니다.

### DISCUSSION THREAD

**Discussion by rhkddud0726** [Lastorder-DC/chatreader-kor#71] [문의] [answered]
안녕하세요. 현재 윈10에 크롬으로 사용 중입니다.

정상 작동하긴 하는데 읽는 소리가 다소 큰거 같아서 낭독프로그램 상으로 조절이 가능한지 궁금합니다.

**Comment by Lastorder-DC** [comment, marked as answer]
채팅창에 `!!tts volume 숫자`를 입력해 설정하실수 있습니다. 숫자는 0에서 100 사이입니다.
예를 들어 80으로 설정시 `!!tts volume 80`을 입력합니다.

스트리머나 모더레이터만 명령을 사용할 수 있습니다. 추후 웹페이지에서 바로 설정/확인 가능하게 수정 예정입니다.
